internal/provider/openstack/firewaller.go

```diff
@@ -554,8 +554,8 @@ func (c *neutronFirewaller) IngressRules(ctx context.ProviderCallContext) (firew
 // OpenModelPorts implements Firewaller interface
 func (c *neutronFirewaller) OpenModelPorts(ctx context.ProviderCallContext, rules firewall.IngressRules) error {
 	err := c.openPortsInGroup(ctx, c.jujuGroupName(c.environ.controllerUUID), rules)
-	if errors.IsNotFound(err) && !c.environ.usingSecurityGroups {
-		logger.Warningf("attempted to open %v but network port security is disabled. Already open", rules)
+	if errors.Is(err, errors.NotFound) && !c.environ.usingSecurityGroups {
+		logger.Infof("attempted to open %v but network port security is disabled. Already open", rules)
```

Comment:
todo: Use Debug instead. I'm wondering how noisy this message is at Warning or Info. Nor am I sure the message adds anything for the end user.